store.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Karma Store</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style2.css">
+</head>
+
+<body>
+    <div class="contener">
+        <div class="nav1">
+            <nav>
+                <img src="img/karma-logo.svg" class="logo">
+                <ul class="nav_bar">
+                    <a href="index.html" class="home"> <strong>Meet Karma</strong></a>
+                    <a href="#" class="home">How it Works</a>
+                    <a class="home" id="active" href="#" class="home">Store</a>
+                    <a href="#" class="home">Blog</a>
+                    <a href="#" class="home">Help</a>
+                    <a href="#" class="home">Login</a </ul>
+            </nav>
+        </div>
+        <div class="form1">
+
+            <h1>Order your Karma wifi device today!</h1>
+            <form>
+                <div class="f">
+                    <label for="username1">First name *</label>
+                    <label for="username2">Last name *</label>
+                </div>
+
+                <div class="f1">
+                    <input id="username1" type="text" name="username" />
+
+                    <input id="username2" type="text" name="username" />
+                </div>
+
+                <label for="address" class="text_ad">Address *</label>
+                <input id="address" type="text" name="address" />
+
+                <label for="address" class="text_ad">Address 2 *</label>
+                <input id="address" type="text" name="address2" />
+
+                <div class="form_c_p">
+                    <label for="city" class="city5">City *</label>
+                    <label for="postcode" class="post5">Postcode *</label>
+                </div>
+
+                <div class="input_c_p">
+                    <select id="city" name="city">
```

Comment:
I think you need a label for your select tag, This text should be the lable `Select your city...` the reason I say this is becaseu if your user don't not select any of your options , the option that says `Select your city...`  could be selected as a default and send to the server which you don't want it to happen. Instead make the first option a blank like this `<option value=""></option>`